code/game/gamemodes/scoreboard.dm

```diff
@@ -663,6 +663,8 @@
 		if(50000 to INFINITY)
 			score["rating"] = "Nanotrasen's Finest"
 	dat += "<B><U>RATING:</U></B> [score["rating"]]<br><br>"
+	send2mainirc("Final crew score: [score["crewscore"]]. Rating: [score["rating"]]")
```

Comment:
If you want it to be a single message, why don't you just put the whole message here? You should still be able to grab the mode and dead players from it

Reply:
Fixed